.gitlab-ci.yml

```diff
@@ -324,3 +330,15 @@ publish:production:s3:scripts:install-mender-sh:
   only:
     refs:
       - production
+
+publish:s3:mender-monitor:
+  only:
+    - /^(master|[0-9]+\.[0-9]+\.x)$/
```

Comment:
This doesn't make sense in this context, as `mender-dist-packages` repository does not have release branches. We can only publish from master - the version information is given by `MENDER_MONITOR_VERSION`, so it should be fine

Reply:
Lol, yes. of course :see_no_evil: